Tools/MonoGame.Tools.Tests/FbxImporterTests.cs

```diff
@@ -93,7 +93,7 @@ public void Dude()
 #if XNA
             Assert.AreEqual(11433, meshContent.Positions.Count);
 #else
-            Assert.AreEqual(13126, meshContent.Positions.Count);
```

Comment:
How did this change @dellis1972 ?  What happened?

Reply:
We bumped assimp. I haven't had the time to dig into why we now get less "Positions" back.

Reply:
I would suspect some weld tolerance or check for invalid triangles was change in AssImp.

Maybe we should double check this before merging?